### PR TITLE
Dark / Light theme documentation

### PR DESCRIPTION
**Pull request by khalilcodes**
## Documentation for dark and light theme modes
#56 
***
This PR adds user and developer documentation for dark/light mode usage or modification.

### Tasks
- [x] Add documentation @ [docs/theme](https://deploy-preview-106--spectacular-dragon-c1015c.netlify.app/docs/theme)
- [x] Update docs with latest config setup - default mode and button icon

### CODE REVIEW

File: site/content/docs/theme.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Users
+
+Flowershow supports the feauture to switch themes in your website to dark or light modes. This can be found in the Navigation bar on the top right hand corner.
+
+The theme by default is set to dark mode and can be switched on/off to change modes.
+
+### Dark theme
+
+![[dark-theme.jpg]]
```
Comment (olayway) [inline site/content/docs/theme.md:11]: see my comment to #107 re images

**Review by olayway — CHANGES_REQUESTED** [review]

**Comment by khalilcodes** [comment]
@olayway I have updated the docs with the changes specified

File: site/content/docs/theme.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <ThemeProvider attribute="class" defaultTheme="dark">
+    ...
+    </ThemeProvider>
+  )
+}
+```
+
+For additional configuration, please read their documentation.
+
+The theme switcher button can be found in `templates/default/components/ThemeSelector.js` file. There you can find the button's icon (svg) and can replace or modify it as per your need.
```
[the remark this answers is not in the document]
Comment (olayway) [reply site/content/docs/theme.md]: Ahhh yes, I see what you mean. In this case, let's maybe just use a default one for now and don't allow for customizing it? I think it is really minor and the icon you chose is really nice.

**Review by olayway — APPROVED** [review]
Thank you @khalilcodes! This is ready to go!

**Pull request merged**